testSetup.js

```diff
@@ -0,0 +1,6 @@
+import register from 'ignore-styles';
+register(['.css', '.sass', '.scss']);
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```

Comment:
Please put an extra line at the end of this file.

Reply:
Okay. I have done this.